[2024-11-25T08:46:45.133+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-25T08:46:45.164+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_PipelineV30.preprocessData scheduled__2024-11-25T07:00:00+00:00 [queued]>
[2024-11-25T08:46:45.180+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_PipelineV30.preprocessData scheduled__2024-11-25T07:00:00+00:00 [queued]>
[2024-11-25T08:46:45.181+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-11-25T08:46:45.203+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): preprocessData> on 2024-11-25 07:00:00+00:00
[2024-11-25T08:46:45.249+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4178) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-25T08:46:45.247+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_PipelineV30', 'preprocessData', 'scheduled__2024-11-25T07:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/etlDag.py', '--cfg-path', '/tmp/tmpzytwpe52']
[2024-11-25T08:46:45.251+0000] {standard_task_runner.py:72} INFO - Started process 4188 to run task
[2024-11-25T08:46:45.252+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask preprocessData
[2024-11-25T08:46:45.331+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_PipelineV30.preprocessData scheduled__2024-11-25T07:00:00+00:00 [running]> on host 7939e06e2b4b
[2024-11-25T08:46:45.478+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Oshith Roshantha' AIRFLOW_CTX_DAG_ID='etl_PipelineV30' AIRFLOW_CTX_TASK_ID='preprocessData' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-25T07:00:00+00:00'
[2024-11-25T08:46:45.481+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-25T08:46:45.587+0000] {python.py:240} INFO - Done. Returned value was:            id                                place  ...      lat   depth
0  us6000p7jc             61 km SE of Shima, Japan  ...  33.8715  339.73
1  us6000p7j5  80 km SE of Katsuren-haebaru, Japan  ...  25.8865   10.00

[2 rows x 13 columns]
[2024-11-25T08:46:45.827+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-25T08:46:45.829+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_PipelineV30, task_id=preprocessData, run_id=scheduled__2024-11-25T07:00:00+00:00, execution_date=20241125T070000, start_date=20241125T084645, end_date=20241125T084645
[2024-11-25T08:46:45.914+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-25T08:46:45.963+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-25T08:46:45.967+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
